src/journey-maps/angular/journey-maps.interfaces.ts

```diff
@@ -74,45 +74,35 @@ export interface SbbBoundingBoxOptions {
 }
 
 /**
- * **WARNING:** The map currently doesn't support more than one of these fields to be set at a time.
+ * **WARNING:** The map doesn't support more than one of these fields to be set at a time.
```

Comment:
@andrinsibold diese Warning ist noch zu unpräzise (war sie vorher auch schon). Müsste so etwas sein wie "The map doesn't support "trip" and "routes" to be set at the same time".